src/renderers/dom/client/syntheticEvents/SyntheticEvent.js

```diff
@@ -111,14 +114,17 @@ assign(SyntheticEvent.prototype, {
   stopPropagation: function() {
     var event = this.nativeEvent;
     if (__DEV__) {
-      warning(event,
-        'This Synthetic event is reused for performance reasons. If you\'re seeing this, ' +
-        'you\'re calling `stopPropagation` on a released/nullified Synthetic event. ' +
-        'This is a no-op. See https://facebook.github.io/react/docs/events.html#event-pooling ' +
-        'for more information.'
+      warning(
+        event,
+        'This synthetic event is reused for performance reasons. If you\'re ' +
+        'seeing this, you\'re calling `stopPropagation` on a ' +
+        'released/nullified synthetic event. This is a no-op. See ' +
+        'https://fb.me/react-event-pooling for more information.'
       );
     }
-    if (!event) return;
```

Comment:
Yea I saw, was just surprised. Turns out I totally missed part of the ESLint migration to 1.0 guide (where they dropped having a default config enabled). #4746 will handle that.